src/Orc.Controls.Examples/ViewModels/DateTimePickerViewModel.cs

```diff
@@ -21,6 +26,36 @@ public DateTimePickerViewModel()
 
         #region Properties
         public DateTime DateTimeValue { get; set; }
+        public FastObservableCollection<object> AvailableFormats { get; private set; }
+        public object SelectedFormat { get; set; }
+        #endregion
+
+        #region Methods
+        protected override async Task InitializeAsync()
+        {
+            await base.InitializeAsync();
+
+            AvailableFormats = new FastObservableCollection<object>();
+            using (AvailableFormats.SuspendChangeNotifications())
+            {
+                foreach (var cultureInfo in CultureInfo.GetCultures(CultureTypes.AllCultures))
```

Comment:
I'd prefer a helper / extension method that returns a list of formats so we don't have to repeat ourselves.